manifests/journal_remote.pp

```diff
@@ -0,0 +1,37 @@
+# @api private
+# @summary This class manages and configures journal-remote.
+# @see https://www.freedesktop.org/software/systemd/man/journal-remote.conf.html
+#
+# @param package_name
+#  name of the package to install for the functionality
+#
+class systemd::journal_remote (
+  Optional[String[1]] $package_name = undef,
+) {
+  assert_private()
+
+  if $package_name {
+    stdlib::ensure_packages($package_name)
+  }
+
+  service { 'systemd-journal-remote':
+    ensure => running,
```

Comment:
should we also set `enable => true,` to put it into autostart? usually we do that.

Reply:
done